src/components/OrderBookSample.stories.tsx

```diff
@@ -0,0 +1,133 @@
+  findTokenBySymbolOrAddress,
+} from 'storybook/tokenDefaults'
+
+type SampleData = typeof sampleDataSet[0]
+
+interface ProduceOrderBookArgsParams {
+  sampleData: Pick<SampleData, 'asks' | 'bids'>
+  orderbookProps: Omit<OrderBookChartProps, 'data'>
+}
+// normalizing sample data
+// because real data is slightly more complex
+const produceOrderBookProps = ({ sampleData, orderbookProps }: ProduceOrderBookArgsParams): OrderBookChartProps => {
+  const { baseToken, quoteToken } = orderbookProps
+
+  const { asks = [], bids = [] } = sampleData
+  const normalizedRawData: OrderBookData = {
+    asks: asks.map(({ volume, price }) => ({
+      // turn small numbers into wei or decimal-aware units
+      // otherwise get filtered as too small by src/components/OrderBookWidget.tsx#L77
+      volume: volume * 10 ** baseToken.decimals,
+      price,
+    })),
+    bids: bids.map(({ volume, price }) => ({
+      volume: volume * 10 ** quoteToken.decimals,
+      price,
+    })),
+  }
+
+  const processedData = processRawApiData({ data: normalizedRawData, baseToken, quoteToken })
+
+  return { ...orderbookProps, data: processedData }
+}
+
+type SampleProps = SampleData & {
+  baseToken: string
+  quoteToken: string
+  networkId: number
+}
+
+// preprocessing template
+// does what OrderBookWidget does for real data
+const SampleTemplate: React.FC<SampleProps> = ({
+  name,
+  description,
+  asks,
+  bids,
+  networkId,
+  baseToken: baseTokenSymbolOrAddress,
+  quoteToken: quoteTokenSymbolOrAddress,
+}) => {
+  const baseToken = useMemo(() => findTokenBySymbolOrAddress(baseTokenSymbolOrAddress, defaultBaseToken), [
+    baseTokenSymbolOrAddress,
+  ])
+
+  const quoteToken = useMemo(() => findTokenBySymbolOrAddress(quoteTokenSymbolOrAddress, defaultQuoteToken), [
+    quoteTokenSymbolOrAddress,
+  ])
+
+  const props = useMemo(
+    () =>
+      produceOrderBookProps({
+        sampleData: { asks, bids },
+        orderbookProps: {
+          baseToken,
+          quoteToken,
+          networkId: networkId,
+        },
+      }),
+    [networkId, baseToken, quoteToken, asks, bids],
+  )
+
+  return (
+    <>
+      <details style={{ lineHeight: 1.5 }}>
+        <summary>{name}</summary>
+        {description}
+      </details>
+      <OrderBookChart {...props} />
+    </>
+  )
+}
+
+/**
+ * {
+ *   Mainnet: "1",
+ *   Ropsten: "3",
+ *   Rinkeby: "4",
+ *   Goerli: "5",
+ *   Kovan: "42"
+ * }
+ */
+const NetworkRadioOptions = Object.entries(Network).reduce<Record<string, string>>((acc, [key, val]) => {
+  if (!Number.isInteger(val)) return acc
+
+  // leaving as number causes radios to lose visible :checked state
+  acc[key] = String(val)
+  return acc
+}, {})
+
+const stories = storiesOf('OrderBook/Sample Data', module)
+stories.addDecorator((Story) => <div style={{ height: '95vh' }}>{Story()}</div>)
+stories.addDecorator(withKnobs)
+stories.addParameters({
+  component: SampleTemplate,
+  docs: { disable: true }, // docs break render
```

Comment:
that's odd

Reply:
that it is
some `locationsMap` internally becomes undefined for some reason
feel free to try it out

Reply:
no worries, these docs are not a big concern right now, maybe if we end up doing a reusable library we can go back to this (and to document everything a bit better). For now, I think it's fine